### PR TITLE
NEWS: add release notes for >v3.0.0 releases

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/6049.

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @xiang90 @heyitsanthony 

Will add `v3.0.5` NEWS when we do the release.

File: NEWS [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  Bug fixes
+
+    - Fix SRV lookup for already-initialized members
+    - Fix raft for leader to correctly set RecentActive of followers
+    - Back-port gRPC upstream patch to fix TLS client issue
+
+  Other changes
+
+    - Revert Dockerfile to use CMD, instead of ENTRYPOINT, to support etcdctl run
```
Comment (xiang90) [inline NEWS]: i think we only needs to include important user facing stuff like this in this doc. for bug fixes, and others, they are already covered in release notes.

File: NEWS [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    - v3 etcdctl default endpoints are now 127.0.0.1:2379
+
+  Backward compatibility
+
+    - Compatible with all previous v3.0.0+ releases
+    - Docker commands for v3.0.2 won't work without specifying executable binary paths
```
Comment (xiang90) [inline NEWS]: we need to keep this too.

File: NEWS (hunk 1 above)
Comment (gyuho) [inline NEWS]: You are right. I will simplify these more.

**Comment by gyuho** [comment]
@xiang90 Just kept changes that directly affect user interface.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**